### ISSUE TITLE
Can't watch nothing on it.

### ISSUE THREAD

**Issue by ruanpereira** [justchokingaround/jerry#12]
I just got a new arch linux, and after installing and trying to watch something, it won't start a episode.
Reinstalled, changed some things in the conf file, in the cache, and nothing worked. 
![Screenshot_20230326_130433](https://user-images.githubusercontent.com/77154574/227788683-2e0cf374-a115-4f7c-a298-99fa14334052.png)

Any ideas on how to fix please? Someone has the same problem or its just me?

**Comment by grigoriynikolaev** [comment]
Hi, same here. Have you tried to run it in incognito mod(jerry -i)?

**Comment by justchokingaround** [comment]
```sh
sh -x $(which jerry) -c 2> idk.log
token=$(cat idk.log | sed -nE "s@.*access_token=(.*)@\1@p" | head -1)
sed -i "/${token}/d" idk.log
```

could you please send me the logs?

**Comment by justchokingaround** [comment]
likely a duplicate of #13

**Comment by ruanpereira** [comment]
> Hi, same here. Have you tried to run it in incognito mod(jerry -i)?

Hii. Just tried, stays the same for me.
![Screenshot_20230326_160118](https://user-images.githubusercontent.com/77154574/227798098-eb3546d1-8067-4115-bcc3-a220727a5b2a.png)

**Comment by justchokingaround** [comment]
logs por favor

**Comment by ruanpereira** [comment]
> likely a duplicate of #13

yeah, just saw the other issue. Gonna send the logs soon. My pc isn't helping lol.

**Comment by ruanpereira** [comment]
> logs por favor
![Screenshot_20230326_165333](https://user-images.githubusercontent.com/77154574/227801013-a6598d99-c3d5-4f09-bb0b-bf9fbe928bf3.png)
40 minutos nisso e não saiu nada ainda. Eu testo a solução em #13 ?

**Comment by justchokingaround** [comment]
just spam enter until the program terminates, that way i can get some logs for what's happening

**Comment by ruanpereira** [comment]
here: https://gist.github.com/ruanpereira/1a4aaf6e8690c9a75e1b1e3f7731dce2

**Comment by justchokingaround** [comment]
https://github.com/justchokingaround/jerry/commit/6a08fb477cf95d81e7b25aea22c6c12ab84d3d99

should fix it, try again after updating jerry with `jerry -U`

**Comment by ruanpereira** [comment]
![Screenshot_20230326_173409](https://user-images.githubusercontent.com/77154574/227803168-592a61a6-f9eb-44de-a887-9f09d74bb0ac.png)
it worked just for one piece(???) and after that goes into that again.

**Comment by justchokingaround** [comment]
logs :point_right: :point_left: pls

**Comment by ruanpereira** [comment]
[idk.log](https://github.com/justchokingaround/jerry/files/11073148/idk.log)

here. While the script was running, the episode started, and after i close it, the log file was created.

**Comment by justchokingaround** [comment]
i need it for an anime that doesn't work

**Comment by ruanpereira** [comment]
![Screenshot_20230326_185854](https://user-images.githubusercontent.com/77154574/227807283-35a2b40f-2796-40ea-b609-428cbdd45566.png)
![Screenshot_20230326_185931](https://user-images.githubusercontent.com/77154574/227807284-c36521d8-09bd-4909-b628-26ac253a4032.png)

so, this is what is hapenning. it won't work, but for some reason whem i run the script this thing starts at one piece only. (one piece is the first one on my watch list)


[idk.log](https://github.com/justchokingaround/jerry/files/11073180/idk.log)

**Comment by justchokingaround** [comment]
just use ur down arrow or `ctrl-j` to select the next entry, same way u would normally do. these commands redirect the output, so that's why u don't see anything on the screen, but it behaves the same as it normally does. so run the commands, then press down arrow, and then press enter

**Comment by ruanpereira** [comment]
[idk.log](https://github.com/justchokingaround/jerry/files/11073446/idk.log) see if it helps

**Comment by justchokingaround** [comment]
the problem is that there is only one sub file passed, but `--sub-files` is used in mpv instead of `--sub-file`. this is a bit stupid, but that's the bug. i deleted ur last logs bc u accidentally shared ur anilist token inside of it @ruanpereira

**Comment by ruanpereira** [comment]
> the problem is that there is only one sub file passed, but `--sub-files` is used in mpv instead of `--sub-file`. this is a bit stupid, but that's the bug. i deleted ur last logs bc u accidentally shared ur anilist token inside of it @ruanpereira

no problem lol

**Comment by justchokingaround** [comment]
@ruanpereira this is fixed now, as i released v1.9.0 of the script (basically a beta version for v2.0.0), the default provider is now 9anime, as it seems to be the most stable. i have not yet documented everything properly, but the `-h` and `--help` argument should give you an idea of what the script can do atm. everything is functional from my testing. sorry for the delay :pensive:

**Comment by ruanpereira** [comment]
wow, thanks! great project my guy. hope you are alright, since it took some time, i imagine that something maybe happened. gonna try the new version later
